requirements.txt

```diff
@@ -0,0 +1 @@
+graphviz==0.20.3
```

Comment:
Should this file add into install folder ?  (If so, need update the install-target to include it).

Reply:
Feedback from Sergey is that it is better to have explicit version and maintain it up-to-date manually.